io.github.CatalystDevOrg.Catalyst/io.github.CatalystDevOrg.Catalyst.appdata.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.CatalystDevOrg.Catalyst</id>
+  
+  <name>Catalyst</name>
+  <summary>A lightweight and user-focused web browser powered by Electron</summary>
```

Comment:
"lightweight" is kind of a lie if it's Electron.

Reply:
That's how it is presented in the developer repository